### DISCUSSION TITLE
Cracks and Pops on client even at the highest latency

### DISCUSSION THREAD

**Discussion by franknitty69** [apohl79/audiogridder#1088] [Help] [answered]
I am trying out audiogridder. I installed the server on a windows 11 desktop. I installed the client on a MacBook pro. When loading the plugin, I immediately get cracks and pops. So much that you cannot hear the original audio. This is without selecting a remote plugin. The same occurs on Maschine, Ableton Live and Logic Pro X.

CPU Load on the server is on 10%, Processing time avg is 0.44 ms. Network I/O is 2.52 MB/s and 1.33 MB/s.

Any suggestions?

**Server**
Windows 11
AMD Ryzen Threadripper 3960X 24-Core
RAM 128 GB
Storage: SSD
10G NIC
AudioGridderServer_1.2.0-beta15.exe

**Client**
2019 MacBook Pro OSX 13
RAM 32GB
Storage: SSD
Wifi AC @ 5GHz
AudioGridderServer_1.2.0-beta15_macOS-x86_64.pkg

**Network**
Ubiquiti AP AC @ 5GHz
Ubiquiti 10G Switch
Ubiquiti Router

**Comment by apohl79** [comment]
Please try to disable "Fixed buffers" in the plugin server/buffer settings and see if that helps.

**Reply by franknitty69** [reply]
fixed buffers did have any effect. same issue.

i did try 1.1.1 on the server and client. default I got cracks and pops, but they went away if I increased the buffer size to 16 blocks (170ms).

**Comment by rncwalker** [comment, marked as answer]
Your not by chance using the Wifi are you?, if you are, that may be your problem. use a hard wire connection.

Also does " This is without selecting a remote plugin" mean the noise is happening even before you select a FX?

Robert

**Reply by franknitty69** [reply]
on my client (macbook pro) i am using wifi. and test the noise occurs as soon as the plugin (vst or au) is added to a track and i have not selected an fx.

i will try with a hard wire. i wanted to test wirelessly to see what the performance would be on 802 ac wifi.

**Comment by franknitty69** [comment]
oh wow. its like night and day using GB networking. I'm getting less than 1 ms performance on 16 plus tracks of SSL channel.
